api/test/wfl/tools/parallel_runner.clj

```diff
@@ -0,0 +1,22 @@
+(ns wfl.tools.parallel-runner
+  (:require [clojure.test :as test]))
+
+(defn -main
+  [& args]
+  (letfn [(go! [& tests]
+            (let [output (new java.io.StringWriter)]
+              (binding [test/*test-out* output
+                        test/*report-counters* (ref test/*initial-report-counters*)]
+                (doseq [t tests] (t))
+                [@test/*report-counters* (str output)])))]
+    (let [ns      (symbol (first args))
+          _       (use ns)
+          tests   (group-by (comp #(contains? % :parallel) meta)
+                            (filter (comp :test meta) (vals (ns-interns (the-ns ns)))))
+          futures (doall (map #(future (go! %)) (tests true)))
+          results (apply list (apply go! (tests false)) (map deref futures))
+          summary (assoc (apply merge-with + (map first results))
+                    :type :summary)]
```

Comment:
The `:type :summary` thing is what lets the `clojure.test/do-report` function print out a nice summary. I could also roll that myself though

Reply:
Or maybe I should just document/comment this?

Reply:
Is this creating one thread for each test marked `^:parallel`? Using an executor here might be advantageous, but maybe for another change.

Reply:
Yeah, I figure that's another change. Figured I'd leave it to the JVM or host to manage the threads since they just spend the whole time sleeping anyway